lnbits/app.py

```diff
@@ -240,7 +238,8 @@ async def check_installed_extensions(app: FastAPI):
                 )
         except Exception as e:
             logger.warning(e)
-            await deactivate_extension(ext.id)
+            settings.deactivate_extension_paths(ext.id)
```

Comment:
wrong

Reply:
reverted, is think the `deactivate_extension` is unecessary its just those 2 lines and used twice, i thought its easier to read and the the actual db action behind it immediatly. but i dont mind...